hash/bucketer.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This file contains the utilities to make bucketing decisions.
+
+package hash
+
+import (
+	"sync"
+
+	lru "github.com/hashicorp/golang-lru"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"knative.dev/pkg/reconciler"
+)
+
+var _ = (reconciler.Bucket)(nil)
```

Comment:
As written this does nothing 🤣 
```suggestion
var _ reconciler.Bucket = (*BucketSet)(nil)
```

Reply:
duh thanks

Reply:
Done